src/tools/compiletest/src/common.rs

```diff
@@ -118,6 +118,21 @@ impl fmt::Display for PassMode {
     }
 }
 
+/// Step at which RMC test should fail.
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd)]
+pub enum RMCFailStep {
+    /// RMC panics before the codegen step. This step runs the same checks on
```

Comment:
Maybe this will make it clearer?
```suggestion
    /// RMC panics before the codegen step (up to MIR generation). [...]
```